hfaerber/Refactor-tractor-Fit-lit
# Miscellaneous PR addressing several small issues

#### What's this PR do?

This PR does several small corrections to close issues.  Unfortunately they are all grouped here as none were large enough on their own to merit a PR request.   

#### Where should the reviewer start?

I apologize for so many little changes to so many different files in one commit.  I will try to outline each change so they are easier to read.   

Start by looking at the issues below.  All have been resolved by this PR.   

This PR also includes a layout for a media query for large screens.   One test was added to our utility class test suite to test hydration data to confirm that rounding conditional was correctly formatted.

If you look at each change in each file you can see the minor changes pretty clearly.   The most significant change is refactoring all the week methods to find the week based on today's date.  

#### How should this be manually tested?

If you pull this branch to your machine, you can see that all tests are still passing and the app is still working correctly.   *There is an issue with huge numbers showing in the activity column.  We believe this is due to changes to the data set by other groups' posts rather than an issue with our methods.*

#### Any background context you want to provide?

I know there is alot going on in this PR and am happy to walk anyone through it if its confusing.  Also happy to assist with any merge conflict that could arise.  

#### What are the relevant tickets?

Issue #54  -  Rounds to decimal or whole number based on which metric is passed
Issue #53  - Week methods have been refactored in all classes except Activity/ActivityRepo/Hydration
Issue #30 Rename variable in SleepRepo more semantically *closed
Issue #50 Rounding the display of the user with the most sleep
#29 Semantically naming the HTML
